flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+    "only-arches": [
+        "x86_64"
```

Comment:
now that is disappointing. I thought the idea Electron is to be cross-platform.

Reply:
Unfortunatly, the server relies on a lib that do not support ARM. So I need to disable support for that arch